rfc/rfc021-vp_token-grant-type.md

```diff
@@ -171,13 +172,14 @@ The following example shows a request to the Presentation Definition endpoint:
 The Authorization Server MAY support an access token introspection endpoint as defined in OAuth 2.0 [RFC7662].
 The introspection endpoint SHOULD map the fields as follows:
 
+* `active`: If the access token is valid.
 * `iss`: The issuer of the access token (DID).
 * `sub`: The subject of the access token (DID), this is the resource owner. Usually the same as the `iss` field.
 * `client_id`: The client (DID) that requested the access token.
 * `exp`: The expiration time of the access token.
 * `iat`: The time the access token was issued.
 * `scope`: The granted scope.
-* `vcs`: The Verifiable Credentials that were used to request the access token using the same encoding as used in the access token request.
+* `vps`: The Verifiable Presentations that were used to request the access token using the same encoding as used in the access token request.
```

Comment:
I think we discussed adding fields specified by input descriptors?

Reply:
both

Reply:
I'd prefer only to include credentials in the response, that were input for the access token, meaning only those matched by the presentation definition. Why do you think we need the whole VP?

Reply:
You shouldn't need any of them if all is done correctly. If we're including any data that was used as input then best to include the raw data which are the vps. The vps will also tell you which wallet signed which vcs. Only including vcs won't give you that info. But we can remove it from the spec and just add them to the implementation?

Reply:
If the `vps` are always the `vp_token` (which is true here), then I think we also need to return the `presentation_submission` to interpret the contents of the `vps`.

Reply:
That's also fine. Might want to choose a style short (acronyms), or long form, though